index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="Home">
+    <div class=" flex nav-bar">
+        <a href="#Home"><img src="./images/gtco_logo2.png" alt="Logo"></a>
+        <ul class="flex nav-list-item">
+            <li class="nav-item nav-about"><a href="about.html">About</a></li>
+            <li class="nav-item nav-color"><a href="#">Exhibition</a></li>
+            <li class="nav-item nav-color"><a href="#">Join</a></li>
+            <li class="nav-item nav-color"><a href="#">Sponsor</a></li>
+            <li class="nav-item nav-color"><a href="#">Schedules</a></li>
+            <li class="nav-item nav-border"><a href="#program">THE CHEFS</a></li>
+        </ul>
+    </div>
+    </header>
+    
+    <div class="bg-wrap">
+    
+    <!-- Navigation Menu Mobile -->
+        <nav class="mobile-menu">
+            <a>
+                <button class="openbtn">
+                    <div class="bar1"></div>
+                    <div class="bar2"></div>
+                    <div class="bar3"></div>
+                </button>
+            </a>
+
+            <div id="menu-list" class="nav-list">
+            <a class="closebtn" href="">&times;</a>
+                <a class="mobile-nav" href="about.html">About</a>
+                <a class="mobile-nav" href="about.html">Contact</a>
+                <a class="mobile-nav" href="#featured-speakers">The Chefs</a>
+                <a class="mobile-nav" href="#">Exhibition</a>
+            </div>
+        </nav>
+    
+      <main id="main">
+    <div class="landing">
+      <p class="before-h12">"Hello! Food Lovers"</p>
+      <h1>GTCO ANNUAL FOOD AND DRINK FESTIVAL 2022 </h1>
+      <p class="description-container">
+        GTCO Food & Drink Fair is a an annual food exhibition and sales event aimed at projecting the diverse angles of the food industry.
+      </p>
+      <h2>2022.04.30 (Wed) ~ 2ND MAY (Sat) <br><p class="paragraph">@ GT Centre Plot 1, Water Corporation Drive, Oniru, VI, Lagos.</p></h2>
```

Comment:
I suggest you activate the transparent background for the mobile version too for consistency in your project. Currently your the mobile version has no transparent background
This is your desktop version with the transparent background:
<img width="1321" alt="Screenshot 2022-10-29 at 17 11 25" src="https://user-images.githubusercontent.com/48546992/198836386-326de260-bb90-4840-b7a7-c4e3f62a42a4.png">
This is your mobile version 
<img width="781" alt="Screenshot 2022-10-29 at 17 11 20" src="https://user-images.githubusercontent.com/48546992/198836402-acd06a7f-2a2f-48d4-8df5-621b19959621.png">

Please note as per the design, they both have a transparent background:
<img width="933" alt="Screenshot 2022-10-29 at 17 11 01" src="https://user-images.githubusercontent.com/48546992/198836437-5fbebc84-ab7b-4868-bf4b-f35eacdc02a0.png">